src/integrationTest/resources/application.yaml

```diff
@@ -48,4 +48,4 @@ tasks:
 
 encryption:
   enabled: ${ENCRYPTION_ENABLED:false}
-  publicKey: ${ENCRYPTION_PUBLIC_KEY}
+  publicKey: ${ENCRYPTION_PUBLIC_KEY:}
```

Comment:
e2e test are run as part of integration tests.This env var should be set imo https://github.com/hmcts/send-letter-service/blob/master/build.gradle#L74? So do we need to do this?

Reply:
something happened and it stopped working really: master/prs/local integration run via gradle task is not working